sdk/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/gwos/tcg/sdk
 
-go 1.17
+go 1.18
+
+require github.com/rs/zerolog v1.26.1
```

Comment:
`sdk` module should avoid 3rd party deps
there is "github.com/gwos/tcg/sdk/logper"

Reply:
Moved the client out of the SDK for now.